stack.yaml

```diff
@@ -0,0 +1,21 @@
+resolver: lts-7.1
+
+packages:
+- '.'
+- location:
```

Comment:
I'm having error with hackage version, will check, maybe something is wrong with my config